examples/serialblaster/serialblaster_main.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, FAR char *argv[])
   int ret;
   int fd;
   FAR char *devpath;
-  int size = 0;
+  int size = sizeof(s)-1;
```

Comment:
```suggestion
  const int slength  = sizeof(s)-1;
  int size = slength;
```
and replace it throughout the code

Reply:
OK. I wanted to reduce number of changes. sizeof(s)-1 in many places is annoying.

Reply:
It is done